docs/configuration/index.md

```diff
@@ -775,6 +775,11 @@ will likely change in the future.
 We will be happy for any feedback regarding it, [please contact us](https://github.com/packit/packit/discussions/2371)!
 :::
 
+#### csmock_args
+
+(*string*) An option to set arguments of `csmock` (the tool used inside [OpenScanHub](https://openscanhub.fedoraproject.org/)). E.g., you can specify `--cppcheck-add-flag=--enable=style` to check styling issues as seen in [this example run](https://openscanhub.fedoraproject.org/task/9993/) triggered on Avahi with [this pull request change](https://github.com/avahi/avahi/pull/635) being reverted.
```

Comment:
```suggestion
(*string*) An option to set arguments of `csmock` (the tool used inside [OpenScanHub](https://openscanhub.fedoraproject.org/)). For instance, you can specify arguments like `--cppcheck-add-flag=--enable=style` to check styling issues as seen in [this example run](https://openscanhub.fedoraproject.org/task/9993/) triggered on Avahi with [this pull request change](https://github.com/avahi/avahi/pull/635) being reverted.
```

and maybe it is just me, but the wording of the last part is not very clear to me

Reply:
You may use this test pull request as an example: https://github.com/avahi/avahi/pull/641

Reply:
@lbarcziova Situ wants to make clear this is not an existing problem of Avahi that we want to blame...;)

Reply:
@lachmanfrantisek That comment was directed towards you to use that pull request as an example, instead of mentioning the revert.

Reply:
(Suggestion applied to both places..)